UI/PauseScreen.cpp

```diff
@@ -449,11 +449,11 @@ UI::EventReturn GamePauseScreen::OnExit(UI::EventParams &e) {
 	// However, let's make sure the config was saved, since it may not have been.
 	g_Config.Save("MainScreen::OnExit");
 
-#ifdef __ANDROID__
-#ifdef ANDROID_NDK_PROFILER
-	moncleanup();
-#endif
-#endif
+	#ifdef __ANDROID__
+	#ifdef ANDROID_NDK_PROFILER
+		moncleanup();
+	#endif
+	#endif
```

Comment:
We do not prefer indenting preprocessor directives.  That said, this block can probably be removed anyway.

-[Unknown]